tests/ui/elements/text.ts

```diff
@@ -0,0 +1,19 @@
+import CustomElement from './custom-element';
+
+export default class Text extends CustomElement {
```

Comment:
We already have TexBox wrapper. for what purpose is this one? Just extend the functionality.
From my point of view, this class is redundant.